tidb-cloud/tidb-cloud-support.md

```diff
@@ -5,17 +5,83 @@ summary: Learn how to contact the support team of TiDB Cloud.
 
 # TiDB Cloud Support
 
-TiDB Cloud offers support services to your organization. To contact our support team, perform the following steps:
+TiDB Cloud offers support services to your organization. You can use the free **Basic** support plan in TiDB Cloud or upgrade to a paid plan for extended services.
 
-1. Click **Support** on the upper right of the top navigation bar, and click **Request Support**. Or directly click the **Help** icon on the lower right.
+The information of each support plan is available on the support page of the TiDB Cloud console. To access the page, see [Check or upgrade your support plan](#check-or-upgrade-your-support-plan).
 
-2. In the **Leave us a message** dialog, fill in the ticket.
+## Request support
 
-3. Click **Send** to submit your ticket. After the support team receives your ticket, they will reply to you via email.
+To request support for your organization or project, perform the following steps:
 
-4. Check your email to view the reply, your ticket history, and communication records.
+1. Click **Support** on the upper right of the top navigation bar, and click **Create New Case**. A **Submit a request** page is displayed.
+
+    > **Note:**
+    >
+    > You can also directly click the **Help** icon on the lower right.
+
+2. On the **Submit a request** page, select **TiDB Cloud Support** from the dropdown list.
+
+3. Fill in the boxes below.
+
+4. Click **Submit** to submit your ticket. After the support team receives your ticket, they will reply to you via email.
```

Comment:
Do we need to mention something like "The response time depends on the support plan of your organization? "

Reply:
```suggestion
4. Click **Submit** to submit your ticket. After the support team receives your ticket, they will reply to you via email. The response time depends on the support plan of your organization.
```